### PR TITLE
Refactor workflows

### PR DESCRIPTION
**Pull request by nikokaoja**
First pass of cleaning up `workflows` module

### CODE REVIEW

File: tests/test_loader.py [hunk 1]
```diff
@@ -12,8 +12,14 @@ def test_load_excel(transformation_rules: TransformationRules):
 
 def test_copy_examples_to_directory(tmp_path: Path):
     target_path = tmp_path / "data"
-
+    print("tmp_path")
```
Comment (doctrino) [inline tests/test_loader.py:15]:
```suggestion
```

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +1,41 @@
...
 COPY cognite/neat /app/cognite/neat
 
 RUN mkdir -p /app/data \
-    && cp /app/cognite/neat/workflows/examples/. /app/data -r \
+    && mkdir -p /app/data/workflows \
+    && mkdir -p /app/data/rules \
+    && mkdir -p /app/data/source-graphs \
+    && cp /app/cognite/neat/workflows/examples/. /app/data/workflows -r \
+    && cp /app/cognite/neat/rules/examples/. /app/data/rules -r \
+    && cp /app/cognite/neat/graph/examples/. /app/data/source-graphs -r \
```
Comment (doctrino) [inline Dockerfile:38]: Are these necessary?  I think the explorer auto copies over when is starts up.

**Review by doctrino — APPROVED** [review]
Primarily moving around files?

File: Dockerfile (hunk 2 above)
Comment (nikokaoja) [reply Dockerfile]: I will test whether this really is needed, I guess not, it is probably left over from where we did not have copy method

**Pull request merged**